### ISSUE TITLE
Quantum infusion cannot be crafted

### ISSUE THREAD

**Issue by joeydawg7** [FTBTeam/FTB-Modpack-Issues#6272]
### Modpack

FTB Presents Direwolf20 1.21

### Modpack version

1.5

### Launcher

FTB App

### Has the pack been modified

No

### Describe the bug

Placing 1 Quantum infused dust into a reaction chamber with 1b of water does not result in Quantum infusion 

### Steps to reproduce

1. Place 1 quantum-infused dust into a reaction chamber with 1 bucket of water
2. nothing happens :(

### Expected behaviour

1b of quantum infusion should be crafted

**Comment by joeydawg7** [comment]
recipe was changed to 4 buckets of water for 1 bucket of quantum infusion. JEI does not properly show recipe

**Comment by ArtpokeBr** [comment]
yeah, can confirm this issue, please make sure to let the dev know by reporting [here](https://github.com/pedroksl/AdvancedAE/issues).
